x-pack/test/cloud_security_posture_functional/pages/findings.ts

```diff
@@ -14,19 +15,48 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const filterBar = getService('filterBar');
   const retry = getService('retry');
   const pageObjects = getPageObjects(['common', 'findings']);
-
-  const data = Array.from({ length: 2 }, (_, id) => {
-    return {
-      resource: { id, name: `Resource ${id}` },
-      result: { evaluation: id === 0 ? 'passed' : 'failed' },
+  const chance = new Chance();
+
+  // We need to use a dataset for the tests to run
+  // We intentionally make some fields start with a capital letter to test that the query bar is case-insensitive/case-sensitive
+  const data = [
```

Comment:
defining the data is now longer and very focused on sorting. i think we can narrow it down a bit. for example, using single chars or shorter sentences would achieve the same result. and instead of doing `chance.integer() % 2 ...` multiple times, we can do it once and do the same for the text casing.